translations/Hindi/general-rules.md

```diff
@@ -0,0 +1,39 @@
+# General Rules for Translation in Hindi
+
+## Overview
+
+* Hindi is a very flexible & simple language. Since it is very much like English, the translations are an easy task.
+
+* Translations can be either formal or informal, but formal translations are preferred for corporate work.
+
+* The translations can be done in any person (first, second or third) according to the content one wants to translate.
+  <br> **Note:** For usage in zulip where we give instructions to the user, it would be recommended to use first person in formal. This will also maintain consistency with the other sites that the user may browse.
```

Comment:
`<br>` is not used in markdown. Use two newlines instead.

Reply:
Updated